vmngclient/api/template_api.py

```diff
@@ -16,6 +17,8 @@
 
 logger = logging.getLogger(__name__)
 
+available_feature_templates: Dict[str, Dict[DeviceModel, FeatureTemplate]] = {}
```

Comment:
What's the usecase of this field?

Reply:
it was for retrieving `FeatureTemplates` for every `DeviceType` & `vManageVersion` in a static way. I'll delete it for now.